src/features/user/forms/PasswordForm.js

```diff
@@ -14,18 +14,28 @@ export const PasswordForm = ({ Component, title, onDone }) => {
     // 2. Listen for those PasSwords from those inputs
     const oldPassword = event.target.elements.currentPasswordInput.value
     const newPassword = event.target.elements.newPasswordInput.value
-
+    const newSameSword = event.target.elements.newSameSwordInput.value
+    // Before calling updateUser make sure the password in both of the new password inputs are the same
     // 3. Do somethings like, send those PasSwords to Daniel's and  ...'s Userbase
-    const { error } = await updateUser({
-      currentPassword: oldPassword,
-      newPassword: newPassword,
-    })
 
-    // 4. Send that customer back to /profile or give alert if error
-    if (error) {
-      alert(error.message)
+    if (newPassword !== newSameSword) {
+        alert(
+          `You typed your "New Password Again" incorrectly, please try again.`
+        )
```

Comment:
**thought (non-blocking):** Could be a good idea to remove content from both fields, so they are forced to write both again.

Reply:
I will create a new issue for this.